issue-pr-templates/bin/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+<!--
+Questions:
+  https://groups.google.com/forum/#!forum/loopbackjs
+  https://gitter.im/strongloop/loopback
+Immediate support:
+  https://strongloop.com/api-connect-faqs/
+  https://strongloop.com/node-js/subscription-plans/
+-->
+
+# Description/Steps to reproduce
+
+<!--
+If feature: A description of the feature
+If bug: Steps to reproduce
+-->
```

Comment:
Thinking out loud and open for discussion... Now that we're more strict about users providing a sandbox for us to reproduce the problem, should we emphasize in here?  something like: 
- if you don't provide this to us, we won't look at it (of course with nicer wording) or expect delays
- And if they think it's very easy to reproduce and there's no need for a sandbox, tell us why.